### DISCUSSION TITLE
Hide current vote results while active

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#744] [Ideas]
_Initially started by Eleven on Canny
June 29, 2021_

Allow the option not to show results of the votes while the vote is still in progress.

Comments from Canny:

1. alix zerd
This is essential for any respectable democratic process

2. Kyle
This is unbelievably important for social communities.  People want to be team players, and personal opinions often get held back in favor of voting along with the "popular" result out of fear of going against the community.  The fact that this isn't a feature yet is a real miss.

**Comment by mariopaolo** [comment]
this is crucial.
I know that it poses an issue about trust, but I believe a UI obfuscation before voting ends is not the end of the world.

**Comment by samwhosung** [comment]
I don't believe this is important until we can hide results with zero-knowledge-proofs. Startkvote is coming to Snapshot in the near future and this will most certainly be a feature.
